core/templates/dev/head/domain/statistics/AnswerDetailsImprovementTaskObjectFactorySpec.ts

```diff
@@ -224,11 +224,15 @@ describe('AnswerDetailsImprovementTaskObjectFactory', function() {
           answer: 'fakeAnswer',
           answer_details: 'fakeAnswerDetails',
           created_on: 1441870501230.642,
-        }]
+        }],
+        _lastUpdatedTime: 1441870501230.642,
+        getLastUpdatedTime: function() {
```

Comment:
Are you sure this is necessary? Constructing a real "LeanerAnswerDetails" object provide this method for free.

Reply:
Sorry should have replied specifically to this comment here. @Seanlip in comment https://github.com/oppia/oppia/pull/8101#issuecomment-569792067 requested I do so. I had erroneously replied in the main thread of comments https://github.com/oppia/oppia/pull/8101#issuecomment-564125391 .
As you can see from there I did as you suggested by using .createDefaultLearnerAnswerDetails(). 
I hope that clears that one up. ;-)